.github/workflows/prep_release.yml

```diff
@@ -0,0 +1,95 @@
+name: Prepare Viash Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release'
+        required: true
+    
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Set up sbt
+      uses: sbt/setup-sbt@v1
+
+    - name: Set up Scala
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y scala
+
+    - name: Build Viash
+      run: |
+        ./configure
+        make
+        make tools
+        bin/viash export json_schema --format=json --output=bin/schema.json
+
+    - name: Get Version
+      id: get_version
+      run: echo "::set-output name=VERSION::${{ github.event.inputs.version }}"
```

Comment:
Here I am fetching the version from the workflow_dispatch input but I could also directly get it from viash such that we need to make sure that the build.sbt is configured correctly before triggering this CI. Wdyt?

Reply:
That is,

```suggestion
      run: |
        VERSION=$(bin/viash --version | cut -d ' ' -f 2)
        echo ::set-output name=VERSION::$VERSION
```